vault_pki.py

```diff
@@ -197,6 +199,21 @@ def _get_vault_connection(config):
     return conn
 
 
+def _send_certs_to_minion(fqdn, dest_path, cert_data):
+    sock_dir = '/var/run/salt/master'
+    cert_path = os.path.join(dest_path, CERT_FILENAME)
+    fullchain_path = os.path.join(dest_path, FULLCHAIN_FILENAME)
+    cert = cert_data['certificate']
+    ca_chain = '\n'.join(cert_data['ca_chain'])
+    fullchain = '\n'.join([cert, ca_chain])
+    cert_info = {"cert": cert, "cert_path": cert_path, "fullchain": fullchain, "fullchain_path": fullchain_path}
+    payload = json.dumps({"data": cert_info})
+    tag = 'request/certificate'
```

Comment:
It is, and I actually went a bit back and forth here (because naming things is hard). The first bit of namespace (`request`) is what is being sent. To that end, I'm replying back to the same namespace listener. If I were to fix it fully, this might turn into `vault_pki/response/certificate`, and change the original request correspondingly.